recipes/tbb/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = "4.4_20160128" %}
+{% set flat_version = "44_20160128" %}
```

Comment:
Jinja let's you do most things you could do in Python. So, `split` and `join` should be options here.